### DISCUSSION TITLE
I can't run my project when I use /app-monorepo

### DISCUSSION THREAD

**Discussion by 3026546679** [OneKeyHQ/app-monorepo#6308] [Q&A]
![企业微信截图_20241205104434](https://github.com/user-attachments/assets/7e8d4d27-4d8c-4257-9e77-c12a5ed364ec)
![企业微信截图_20241205111746](https://github.com/user-attachments/assets/e1e59015-5377-446f-a216-dcb580794c21)

**Comment by 3026546679** [comment]
Am I missing some basic context?

**Reply by irajsava** [reply]
__****__
